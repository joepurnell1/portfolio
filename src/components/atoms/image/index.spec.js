import React from 'react';
import renderer from 'react-test-renderer';
import Image from './index';

describe('image component', () => {
  it('should render default as expected', () => {
    const render = renderer.create(
      <Image
        imageSource="https://static.stereogum.com/uploads/2012/09/pixies-608x451.jpeg"
        imageAlt="The Pixies"
      />,
    );
    expect(render.toJSON()).toMatchSnapshot();
  });

  it('should render with custom sizes as expected', () => {
    const render = renderer.create(
      <Image
        imageSource="https://ksassets.timeincuk.net/wp/uploads/sites/55/2015/08/2013PixiesEM051213.hero_-1-920x515.jpg"
        imageAlt="The Pixies"
        height="600px"
        width="300px"
      />,
    );
    expect(render.toJSON()).toMatchSnapshot();
  });
});
