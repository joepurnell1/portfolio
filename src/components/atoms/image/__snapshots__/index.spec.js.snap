// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`image component should render default as expected 1`] = `
<img
  alt="The Pixies"
  className="sc-bdVaJa bEeBlh"
  height="42px"
  src="https://static.stereogum.com/uploads/2012/09/pixies-608x451.jpeg"
  width="42px"
/>
`;

exports[`image component should render with custom sizes as expected 1`] = `
<img
  alt="The Pixies"
  className="sc-bdVaJa bWCENx"
  height="600px"
  src="https://ksassets.timeincuk.net/wp/uploads/sites/55/2015/08/2013PixiesEM051213.hero_-1-920x515.jpg"
  width="300px"
/>
`;
